communication/moderators.md

```diff
@@ -102,6 +102,12 @@ Regional category moderators seats (per category): 3
 | Ihor Dvoretskyi  | @idvoretskyi          | EMEA   | [EET - Eastern European Time](https://time.is/EET) |
 | *Open*           | *Open*                |        |                                                    |
 
+**[Spanish]**
```

Comment:
Two small things:
- If you don't mind, could you put this section above the Ukraine? That way they're listed in alphabetical order.
- Add a link at the bottom of the doc to here:  https://discuss.kubernetes.io/c/regional-discussions/spanish
  That way it'll mimic the other regional sections and link out to them.

Thanks!

Reply:
Both addressed in  https://github.com/kubernetes/community/compare/8c1748a0d0f186965d7947a8cdcd37e4f29ce29e..d42c6d757200752ce4f42f9b515af2f8aeddb0e4.

I was wondering how the link in the `Language` header was being set, now I know. ^^